source/topics/account/how-to-manage-billing.md

```diff
@@ -0,0 +1,25 @@
+For billing information, navigate to your account settings in the Aptible dashboard and click on Billing. 
+
+1. Under the _Overview_ tab, you can see: 
+
+    - Your current plan and monthly base rate
+    - The number of resources (containers, disk space, and domains) deployed per environment  _(Note: At time of writing, the dashboard does not yet reflect custom [container sizes](https://github.com/aptible/dashboard.aptible.com/issues/516))_
+    - Any billable usage
+    - The unit pricing rate per resource type
+
+2. Under the _Plan_ tab you can view your current plan, or upgrade plans.
+
+3. Under the _Payment Method_ tab you can change or edit your payment method, and view
+
+    - Your current plan
+    - Your account's billing anniversary 
+    - The estimated monthly cost of your current configuration
+    - Your account's current balance
+
+4. To pay by ACH, send a support ticket to support@aptible.com with the following bank information:
+
+    - Account owner name 
+    - Routing Number
+    - Account Number
+
+5. To change the recipient of monthly invoices (automatically sent at time of payment), please [contact support](support@aptible.com).
```

Comment:
That's not a valid link. You want `mailto:support@aptible.com`. Side note: let's remember to change all these mailto links to point to the support request form.